src/openrct2/scripting/ScriptEngine.cpp

```diff
@@ -873,6 +916,51 @@ void ScriptEngine::RunGameActionHooks(const GameAction& action, std::unique_ptr<
     }
 }
 
+static std::unique_ptr<GameAction> CreateGameActionFromActionId(const std::string& actionid)
+{
+    const static std::map<std::string, uint32_t> ActionNameToType = {
```

Comment:
Won't make a huge difference, but we don't need the sorting anyway
```suggestion
    const static std::unordered_map<std::string, uint32_t> ActionNameToType = {
```